WHITEPAPER.md

```diff
@@ -191,9 +191,9 @@ Here is an example state of Transcoder options that a delegator can review when
 
 Transcoders who are open for business on the network, throw their hat into the ring for transcoding work by submitting a `TranscodeAvailability()` transaction. This indicates their availability and places them into a pool of transcoders available to take a newly submitted job.
 
-When a broadcaster submits their stream into the Livepeer network it is given a `StreamID`. This serves as both a unique identifier, and it also contains the origin node address so that nodes know how to request and route requests to consume this stream towards the origin. The stream contains many consecutive `Segments`, as described in the [Video Segments](#video-segments) section. If the broadcaster would like the network to take care of transcoding their stream into all the formats and bitrates necessary to reach every user on every device, then the first step is submitting a transcoding job transaction on chain. A job consists of:
+When a broadcaster submits their stream into the Livepeer network it is given a `StreamID`. This serves as both a unique identifier, and it also contains the origin node address so that nodes know how to request and route requests to consume this stream towards the origin. The stream contains many consecutive `Segments`, as described in the [Video Segments](#video-segments) section. If the broadcaster would like the network to take care of transcoding their stream into all the formats and bitrates necessary to reach every user on every device, then the first step is submitting a transcoding job transaction on chain. Jobs are given a unique ID as well, and job consists of:
 
-`Job(StreamID, TranscodingOptions, PricePerSegment)`
+`Job(JobID, StreamID, TranscodingOptions, PricePerSegment)`
```

Comment:
Small thought - should we just list out the fields in bullets? Writing it this way kind of makes it look like you invoke the job function with a jobId field.

Reply:
Good point. I don't want to represent that this is the entire internal metadata state of an on chain job. I'm going to change it to say that "the input data to job consists of" and remove the JobID param from this sample call. 

It's ok if it's not exact to implementation, as it's more to spirit than technical spec.